Automattic/jetpack
# VideoPress: Add file drop support after first video

Fixes #27153

#### Changes proposed in this Pull Request:

* Adds a file drop wrapper in the Admin area

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Go to the VideoPress dashboard with at least one video already uploaded
* Drag a file over the dashboard area
* Check that it filters incorrect files and uploads at most one file

https://user-images.githubusercontent.com/8486249/200656715-8b43452e-0578-43e2-aa42-22e45111483c.mp4
